### ISSUE TITLE
FZF + Neovim + tmux hiding neovim content while searching

### ISSUE THREAD

**Issue by vinioliveira** [junegunn/fzf#1506]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [ ] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [x ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [x ] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ ] bash
    - [x ] zsh
    - [ ] fish

So i'm not quite sure if it's a neovim or fzf hope you guys can help me on this, I am running into a very anoying issue, for some reason, anytime I run FZF from neovim with let g:fzf_prefer_tmux set as 1 it seems that neovim gets suspended, and blocks all the content of the file opened at the moment, I know it maight seems a very silling problem but sometimes I'd like to search something based an within the file I'm editing. Here's how it looks like. 

![screen recording 2019-02-15 at 09 09 am](https://user-images.githubusercontent.com/341918/52855865-72aa4300-3101-11e9-81a1-6757362b0ff9.gif)

this very simple setup is able to reproduce this issue 

```vim
" init.vim
call plug#begin('~/.vim/plugged')

Plug '/usr/local/opt/fzf'
Plug 'junegunn/fzf.vim'

call plug#end()

let g:fzf_prefer_tmux = 1
```

NVM: 0.3.4
FZF: 0.17.5
TMUX: 2.8

Thanks in advanvance.

**Comment by junegunn** [comment]
Yeah, it's a known issue on Neovim, but I'm not aware of a way to fix it. You can easily reproduce the case using the following command:

```vim
" Hides the upper window if run on Neovim, but not on regular Vim
:call system('echo foo | fzf-tmux')
```

**Comment by denisinla** [comment]
My money is on tmux being the issue. If I have tmux disabled then run fzf in neovim I do not get this issue.

**Comment by alvarosevilla95** [comment]
This is still an issue. Is this related to how neovim launches fzf through the built in terminal instead of as an external command like in regular vim?
